src/components/step/YouTubeStep/YouTubeStep.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react'
+import MyDataHelps, {StepConfiguration} from '@careevolution/mydatahelps-js';
+import StepTitle from '../StepTitle';
+import StepText from '../StepText';
+import StepDetailText from '../StepDetailText'
+import StepNextButton from '../StepNextButton';
+import '../step.css';
+
+export interface YouTubeStepProps {
+    properties?: any;
```

Comment:
I assume these props are entirely for testing...but from what I can see won't actually update the step if you change them (they just set the default value) which might be unexpected for someone using this library.

One thing that I haven't done a great job at (mostly because I learned about the pattern halfway through building this repo) is to separate out presentational vs. container components.  

Given that we want to make things reusable with our outside collaborators, I think it would be worth it to split that out here such that YoutubeStep is *purely* presentational (zero state, just props) and then we have a YoutubeStepContainer component that has the state/useEffect call.

Reply:
Yeah, it makes sense generally and I've gone ahead and split this out. Interestingly, the guy who originally popularized this seems to have semi-disavowed it, or at least encouraged people not to over apply:

https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.5hwzcvetl
https://blog.bitsrc.io/implementing-the-container-pattern-using-react-hooks-f490a8492d05

It definitely makes sense if you have any reason to use the presentational bit outside the original/larger context. I'm not sure that will be the case here with this step though. Outside of the storybook framework itself it does not seem we would use the presentational YouTubeStep without its container in the real world. So would be good to understand how much the storybook framework and making things isolated for presentational *testing* is a driver in your thinking about this.

Reply:
I already split it out, was not a big deal in this case. I just wanted to have a clear rationale in my head for when/why we would implement this pattern or exceptions to it.